solr/schema.xml

```diff
@@ -22,44 +22,49 @@
 -->
 
 <!-- We update the version when there is a backward-incompatible change to this
-schema. In this case the version should be set to the next CKAN version number.
-(x.y but not x.y.z since it needs to be a float) -->
-<schema name="ckan" version="2.8">
+schema. We used to use the `version` attribute for this but this is an internal
+attribute that should not be used so starting from CKAN 2.10 we use the `name`
+attribute with the form `ckan-X.Y` -->
+<!-- With upgrade of ADR to CKAN 2.9.5 we're updating solr config with 
+     https://raw.githubusercontent.com/ckan/ckan/ckan-2.9.5/ckan/config/solr/schema.xml
```

Comment:
That should say: https://raw.githubusercontent.com/ckan/ckan/ckan-2.9.5/ckan/config/solr/schema.solr8.xml